apps/portal/js/ues-dashboards.js

```diff
@@ -109,46 +141,75 @@
      */
     var convertToDesignerLayout = function(json) {
     
-        var componentBoxListHbs = Handlebars.compile($("#ues-component-box-list-hbs").html() || ''), 
+        var componentBoxListHbs = Handlebars.compile($("#ues-component-box-list-hbs").html() || ''),
             container = $('<ul />');
         
-        $(componentBoxListHbs(json)).appendTo(container).html();
+        $(componentBoxListHbs(json)).appendTo(container)
         
         return container;
     }
-
+    
+    /**
+     * renders a page in the dashboard designer and the view modes
+     * @param element
+     * @param dashboard
+     * @param page
+     * @param pageType
+     * @param done
+     * @param isDesigner
+    */
     var renderPage = function (element, dashboard, page, pageType, done, isDesigner) {
+        
```

Comment:
Check the possibility of breaking this function to smaller functions, because too many params